src/document/generator.ml

```diff
@@ -1533,7 +1537,7 @@ module Make (Syntax : SYNTAX) = struct
       let content = { Include.content; status; summary } in
       let attr = [ "include" ] in
       let anchor = None in
-      let doc = Comment.first_to_ir sg_doc in
+      let doc = Comment.synopsis ~decl_doc:[] ~expansion_doc:(Some sg_doc) in
```

Comment:
Unrelated to this PR:
This ignores `t.doc` and takes only the first paragraph of the expansion doc, the rest is removed. I think we should at least show `t.doc` in its entirety.
I'll propose something is an other PR.

Reply:
Issue: https://github.com/ocaml/odoc/issues/645